test/models/member_test.rb

```diff
@@ -3,7 +3,20 @@
 require 'test_helper'
 
 class MemberTest < ActiveSupport::TestCase
```

Comment:
The Algolia search option needs to be enabled for the `test` environment, and we need some coverage on the usage:

<img width="607" alt="code coverage for member-portal 2017-12-02 11-29-27" src="https://user-images.githubusercontent.com/984/33519090-98e85d02-d754-11e7-9266-d4a33e02e28c.png">

You could use Minitest to set up an expectation that `.search` gets called when enabled and return a mock response to prevent the request from being dispatched to Algolia in the test run.